src/Microsoft.DotNet.Build.Tasks/GetPackageVersion.cs

```diff
@@ -34,7 +34,7 @@ public override bool Execute()
                 select el).FirstOrDefault();
 
             Debug.Assert(embeddedVerNode != null, "embeddedVerNode shouldn't be null");
-            PackageVersion = string.Format("{0}-{1}", embeddedVerNode.Value, DailyBuildNumber);
+            PackageVersion = string.Format("{0}-{1}", embeddedVerNode.Value, BuildNumber);
```

Comment:
Does this work in the version in the nuspec is just a version without a text name? something like 1.0.0-00005 with that work? I know we hit the error when it was 0.0.0-4 not sure if that was just because of the 0 version or if it buildnumer requires a spec.